### ISSUE TITLE
[Bug]: game crashes as I join

### ISSUE THREAD

**Issue by Wood075** [typical-developers/oaklands-feedback#709]
### Roblox Username

OP_Godzilla

### What version did this issue occur in?

1.61.1

### Describe the issue in detail.

The game crashed as I join for some reason
Do note that I’m using a phone that works playing the game

### Provide some images / videos of the issue in action.

https://github.com/typical-developers/oaklands-feedback/assets/167630086/8577096c-d5f2-4321-9b54-8c8d41db5efa

### Before Submitting

- [X] I have confirmed that nobody else reported this issue.
- [X] The details of this report are legitimate.
- [X] I understand that I cannot be reimbursed here for in-game content I have lost.

**Comment by DerpTheNerp** [comment]
Yeah same here, sometimes let's me load into the game but then crashes, im on ps5
